onnxruntime/core/common/path_utils.h

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include "core/common/common.h"
+#include "core/common/path_string.h"
+
+namespace onnxruntime {
+
+namespace path_utils {
+
+/** Return a PathString with concatenated args. */
+template <typename... Args>
+PathString MakePathString(const Args&... args) {
```

Comment:
nice. i believe MakeString falls back to using stringstream. does that work properly with wstring arguments?

Reply:
if wstring args are not currently supported, it may be good to add a comment for now indicating that.
it's kind of unexpected to me for MakePathString() to not support PathString arguments (i.e. on Windows).